### PR TITLE
Update indexing documentation for Exponential Rebates

### PR DESCRIPTION
**Pull request by alex-pakalniskis**
Removes references to Cobb-Douglas, rebate pools, and legacy allocation states. Tagging @tmigone for visibility.

Thanks :smile:

### CODE REVIEW

File: website/pages/en/network/indexing.mdx [hunk 1]
```diff
@@ -81,17 +81,17 @@ Disputes can be viewed in the UI in an Indexer's profile page under the `Dispute
 
 ### What are query fee rebates and when are they distributed?
 
-Query fees are collected by the gateway whenever an allocation is closed and accumulated in the subgraph's query fee rebate pool. The rebate pool is designed to encourage Indexers to allocate stake in rough proportion to the amount of query fees they earn for the network. The portion of query fees in the pool that are allocated to a particular Indexer is calculated using the Cobb-Douglas Production Function; the distributed amount per Indexer is a function of their contributions to the pool and their allocation of stake on the subgraph.
+Query fees are collected by the gateway and distributed to indexers according to the exponential rebate function (see GIP here: https://forum.thegraph.com/t/gip-0051-exponential-query-fee-rebates-for-indexers/4162).
```
Comment (benface) [inline website/pages/en/network/indexing.mdx:84]:
```suggestion
Query fees are collected by the gateway and distributed to indexers according to the exponential rebate function (see GIP [here](https://forum.thegraph.com/t/gip-0051-exponential-query-fee-rebates-for-indexers/4162)).
```

File: website/pages/en/tokenomics.mdx [hunk 2]
```diff
@@ -75,7 +75,7 @@ Indexers are the backbone of The Graph. They operate independent hardware and so
 
 Indexers can earn GRT rewards in two ways:
 
-1.  Query fees: GRT paid by developers or users for subgraph data queries. Query fees are deposited into a rebate pool and distributed to Indexers.
+1.  Query fees: GRT paid by developers or users for subgraph data queries. Query fees are directly distributed to Indexers according to the exponential rebate function (see GIP here: https://forum.thegraph.com/t/gip-0051-exponential-query-fee-rebates-for-indexers/4162).
```
Comment (benface) [inline website/pages/en/tokenomics.mdx:78]:
```suggestion
1.  Query fees: GRT paid by developers or users for subgraph data queries. Query fees are directly distributed to Indexers according to the exponential rebate function (see GIP [here](https://forum.thegraph.com/t/gip-0051-exponential-query-fee-rebates-for-indexers/4162)).
```

File: website/pages/en/network/indexing.mdx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Indexing
 ---
 
-Indexers are node operators in The Graph Network that stake Graph Tokens (GRT) in order to provide indexing and query processing services. Indexers earn query fees and indexing rewards for their services. They also earn from a Rebate Pool that is shared with all network contributors proportional to their work, following the Cobb-Douglas Rebate Function.
+Indexers are node operators in The Graph Network that stake Graph Tokens (GRT) in order to provide indexing and query processing services. Indexers earn query fees and indexing rewards for their services. They also earn query fees that are rebated according to an exponential rebate function.
```
Comment (azf20) [inline website/pages/en/network/indexing.mdx:5]: I think it might be worth providing a bit more of a legible explanation of what the rebate function is for (essentially ensuring that indexers have sufficient stake allocated when claiming a given amount of query fees)

**Review by azf20 — APPROVED** [review]
Nice! I think this is good from a "removing prior references" standpoint, I think maybe there could be a short explanation within the docs themselves (vs. always linking to the GIP) to explain why there is a rebate function at all, and in laymans terms how the exponential rebate function works.

**Review by tmigone — APPROVED** [review]
LGTM, agree with Adam's point on adding a brief explainer on the new mechanism, maybe in the section where we link to the GIP we could add one sentence or two kind of a TL;DR for the GIP. @howardheaton might be able to help?

File: website/pages/en/network/indexing.mdx (hunk 3 above)
Comment (howardheaton) [reply website/pages/en/network/indexing.mdx:5]: Agreed. To be explicit, after "to an exponential rebate function.", you could add:

> This aim of using this rebate function is to ensure indexers achieve the best outcome by faithfully serving queries. This works by incentivizing Indexers to allocate a large amount of stake (which can be slashed for erring when serving a query) relative to the amount of query fees they may collect.

**Comment by howardheaton** [comment]
@tmigone, I agree. It's a bit long, but how about this for a tl;dr?

>  The exponential rebates function is proposed as a way to ensure indexers achieve the best outcome by faithfully serving queries. It works by incentivizing Indexers to allocate a large amount of stake (which can be slashed for erring when serving a query) relative to the amount of query fees they may collect.
